powerlevel9k.zsh-theme

```diff
@@ -240,12 +240,12 @@ right_prompt_segment() {
   if [[ -n "$6" ]]; then
     visual_identifier="$(print_icon $6)"
     if [[ -n "$visual_identifier" ]]; then
+      # Add an whitespace if we print more than just the visual identifier
+      [[ -n "$5" ]] && visual_identifier=" $visual_identifier"
```

Comment:
Could you here add a comment about the whitespace as well?

Reply:
Done.